src/index.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component, PropTypes } from 'react';
-import BootstrapEditor from "./bootstrapEditor";
+import BootstrapEditor from "./materialUIEditor";
```

Comment:
I believe that the sole "index.js" is not enough anymore. "index.js" should be duplicated and moved into two separated subfolders: one for "bootstrap" one for "material-ui". Then each specific editor should be moved into the corresponding folder. When done, I'll think I'll make two demo: one with material-ui and one with bootstrap.

Reply:
"index.js" should be duplicated and moved into two separated subfolders: one for "bootstrap" one for "material-ui". Then each specific editor should be moved into the corresponding folder. 

I do understand the Step above (done), but i do not know how to make the website work after that.

Reply:
One folder `demo` is not enough: `demo` should be split into `demo/bootstrap` and `demo/material-ui`. Each with it own `index.html` targetting a specific `bundle.js`.

You'll have to change webpack config [file](https://github.com/damienleroux/react-grid-layout-builder/blob/master/webpack.config.js) to have [multiple bundle.js](https://webpack.js.org/concepts/output/#multiple-entry-points) output in `demo` sub folders

If you have no idea what I'm talking about XD, made a pull request now into the [`develop`](https://github.com/damienleroux/react-grid-layout-builder/tree/develop) branch: I'll merge and will make the required adjustements